packages/image-cropper/README.md

```diff
@@ -0,0 +1,3 @@
+# Image cropper
```

Comment:
Amazing work on migrating to packages 👏🏼

Reply:
Thanks :) It became a bit easier when I found that there were actually two existing packages with code in them (`utils` and `data`) so I had somewhere to start from. I hope that everything in package.json and tsconfig.json makes sense.

Reply:
🙌🏼